packages/icon/test/icon.test.ts

```diff
@@ -69,6 +70,25 @@ describe('Icon', () => {
         await expect(el).to.be.accessible();
     });
 
+    it('loads w/ invalid src, error dispatching', async () => {
+        const error = Sinon.spy();
+        const el = await fixture<Icon>(
+            html`
+                <sp-icon
+                    label="Image Icon"
+                    src="invalid-image-src"
+                    @error=${error}
+                ></sp-icon>
+            `
+        );
+
+        await elementUpdated(el);
+
+        await expect(el).to.be.accessible();
+
+        expect(error).to.be.calledWithExactly(new Event('error'));
```

Comment:
Does WebKit pass on this test for you locally? CI is showing `expected spy to have been called with exact arguments { isTrusted: [Getter] }`. If you thought there was something about the event construction you thought could match that up, that would be great. However, a generic `error.called` or `error.calledOnce` would also be enough here as there's no real information in the error itself.

Reply:
Seems the spy is not getting called at all, looking for a way to check whether the same is supported via Webkit

Reply:
You may also try changing the order of the bindings here. While it would be a bit unexpected, the error could be dispatched synchronously with the erroneous `src` application, which currently happens just before the event binding.

Reply:
Found a bug logged for the same on WebKit's end https://github.com/microsoft/playwright/issues/22332

Reply:
@Westbrook Any suggestion on this, what could be done next if this is only failing for playwright webkit type ones

Reply:
Like I mentioned above, `error.called` or `error.calledOnce` or `error.callCount` are all fine alternatives to `calledWithExactly` In my opinion.

Reply:
But for the playwright webkit, the above suggestions are also failing, I hope we are good without this feature for the said type browser for now

Reply:
I see, I didn't quite catch that. You can leverage the helpers available here: https://github.com/adobe/spectrum-web-components/blob/main/tools/shared/src/platform.ts#L52 to query the browser and `skip` the test just for WebKit. Please be sure to add a comment as to why and link back to that issue.